packages/vite/src/node/optimizer/scan.ts

```diff
@@ -259,7 +259,9 @@ function orderedDependencies(deps: Record<string, string>) {
 function globEntries(pattern: string | string[], config: ResolvedConfig) {
   const resolvedPatterns = arraify(pattern)
   if (resolvedPatterns.every((str) => !glob.isDynamicPattern(str))) {
-    return resolvedPatterns.map((p) => path.resolve(config.root, p))
+    return resolvedPatterns.map((p) =>
+      normalizePath(path.resolve(config.root, p)),
+    )
```

Comment:
Does `glob` automatically normalize the returned paths in Windows? We should probably keep this return to be consistent with it